src/Sylius/Component/Core/Model/OrderItemUnitInterface.php

```diff
@@ -11,16 +11,26 @@
 
 namespace Sylius\Component\Core\Model;
 
-use Sylius\Component\Inventory\Model\InventoryUnitInterface;
+use Sylius\Component\Inventory\Model\StockableInterface;
 use Sylius\Component\Order\Model\OrderItemUnitInterface as BaseOrderItemUnitInterface;
 use Sylius\Component\Shipping\Model\ShipmentUnitInterface;
 
 /**
  * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
  */
-interface OrderItemUnitInterface extends BaseOrderItemUnitInterface, InventoryUnitInterface, ShipmentUnitInterface
+interface OrderItemUnitInterface extends BaseOrderItemUnitInterface, ShipmentUnitInterface
```

Comment:
Why removing `InventoryUnitInterface`? The concept of Inventory unit should stay.